assembl/views/api2/discussion.py

```diff
@@ -453,54 +454,55 @@ def get_time_series_analytics(request):
+                                               func.count(memberAgentStatus.id).label('count_approximate_members')
+                                               )
+        members_subquery = members_subquery.outerjoin(memberAgentStatus, ((memberAgentStatus.last_unsubscribed >= intervals_table.c.interval_end) | (memberAgentStatus.last_unsubscribed.is_(
+            None))) & ((memberAgentStatus.first_subscribed < intervals_table.c.interval_end) | (memberAgentStatus.first_subscribed.is_(None))) & (memberAgentStatus.discussion_id == discussion.id))
         members_subquery = members_subquery.group_by(intervals_table.c.interval_id)
         members_subquery = members_subquery.subquery()
 
         subscribersAgentStatus = aliased(AgentStatusInDiscussion)
         subscribers_query = discussion.db.query(intervals_table.c.interval_id,
-            func.sum(
-                case([
-                      (subscribersAgentStatus.last_visit == None, 0),
-                      (and_(subscribersAgentStatus.last_visit < intervals_table.c.interval_end,
-                        subscribersAgentStatus.last_visit >= intervals_table.c.interval_start), 1)
-                      ], else_=0)
-                     ).label('retention_count_last_visit_in_period'),
-            func.sum(
-                case([
-                      (subscribersAgentStatus.first_visit == None, 0),
-                      (and_(subscribersAgentStatus.first_visit < intervals_table.c.interval_end,
-                        subscribersAgentStatus.first_visit >= intervals_table.c.interval_start), 1)
-                      ], else_=0)
-                     ).label('recruitment_count_first_visit_in_period'),
-            func.sum(
-                case([
-                      (subscribersAgentStatus.first_subscribed == None, 0),
-                      (and_(subscribersAgentStatus.first_subscribed < intervals_table.c.interval_end,
-                        subscribersAgentStatus.first_subscribed >= intervals_table.c.interval_start), 1)
-                      ], else_=0)
-                     ).label('recruitment_count_first_subscribed_in_period'),
-            func.sum(
-                case([
-                      (subscribersAgentStatus.last_unsubscribed == None, 0),
-                      (and_(subscribersAgentStatus.last_unsubscribed < intervals_table.c.interval_end,
-                        subscribersAgentStatus.last_unsubscribed >= intervals_table.c.interval_start), 1)
-                      ], else_=0)
-                     ).label('retention_count_last_unsubscribed_in_period'),
-        )
-        subscribers_query = subscribers_query.outerjoin(subscribersAgentStatus, subscribersAgentStatus.discussion_id==discussion.id)
+                                                func.sum(
```

Comment:
I think you could have fixed the identation warning by just putting a new line just after `discussion.db.query(`